README.md

```diff
@@ -44,9 +46,13 @@ behavior to code-of-conduct@atomist.com.
 ## Documentation
 
 Please see [docs.atomist.com][atomist-doc] for
-[developer][atomist-doc-sdm] documentation.
+[developer][atomist-doc-sdm] documentation and for high-level documentation of [this pack][atomist-doc-k8-pack].
+
+Also see the TypeScript [API Doc][apidoc] for this pack.
```

Comment:
Is the above link necessary if this one exists?

Reply:
I wanted one at the beginning